python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -549,6 +575,8 @@ cdef class ParquetWriter:
                 .stats_level(self.stat_freq)
                 .build()
             )
+            args.set_row_group_size_bytes(self.row_group_size_bytes)
+            args.set_row_group_size_rows(self.row_group_size_rows)
```

Comment:
Right. I started with that but hit this error:
```
Error compiling Cython file:
------------------------------------------------------------
...
        cdef vector[map[string, string]] user_data
        user_data = vector[map[string, string]](num_partitions, tmp_user_data)

        cdef chunked_parquet_writer_options args
        with nogil:
            args = move(
                      ^
------------------------------------------------------------

cudf/_lib/parquet.pyx:570:23: Cannot assign type 'parquet_writer_options' to 'chunked_parquet_writer_options'
Traceback (most recent call last):
  File "setup.py", line 236, in <module>
    setup(
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/__init__.py", line 87, in setup
    return distutils.core.setup(**attrs)
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 148, in setup
    return run_commands(dist)
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
    dist.run_commands()
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
    self.run_command(cmd)
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/dist.py", line 1229, in run_command
    super().run_command(command)
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 985, in run_command
    cmd_obj.ensure_finalized()
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 107, in ensure_finalized
    self.finalize_options()
  File "setup.py", line 148, in finalize_options
    self.distribution.ext_modules = cythonize(
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 1127, in cythonize
    cythonize_one(*args)
  File "/home/rzamora/miniconda/miniconda3/envs/cudf_2208/lib/python3.8/site-packages/Cython/Build/Dependencies.py", line 1250, in cythonize_one
    raise CompileError(None, pyx_file)
Cython.Compiler.Errors.CompileError: cudf/_lib/parquet.pyx
```
I didn't bother investigating, because I knew `args.set_row_group_size_*` would work, but I'll try to figure it out (though suggestions are welcome)

Reply:
It's a typo in the pxd file:

```diff
diff --git a/python/cudf/cudf/_lib/cpp/io/parquet.pxd b/python/cudf/cudf/_lib/cpp/io/parquet.pxd
index d02fffe9c0..d6b587af42 100644
--- a/python/cudf/cudf/_lib/cpp/io/parquet.pxd
+++ b/python/cudf/cudf/_lib/cpp/io/parquet.pxd
@@ -192,10 +192,10 @@ cdef extern from "cudf/io/parquet.hpp" namespace "cudf::io" nogil:
         chunked_parquet_writer_options_builder& compression(
             cudf_io_types.compression_type compression
         ) except +
-        parquet_writer_options_builder& row_group_size_bytes(
+        chunked_parquet_writer_options_builder& row_group_size_bytes(
             size_t val
         ) except+
-        parquet_writer_options_builder& row_group_size_rows(
+        chunked_parquet_writer_options_builder& row_group_size_rows(
             size_type val
         ) except+
 ```